src/components/Account-panel/Contact-info/Contact-info.tsx

```diff
@@ -1,35 +1,96 @@
-import { useEffect, useState } from 'react'
-import { Form, Formik } from 'formik'
-import { Box, Grid, Typography } from '@mui/material'
-import axiosInstance from '../../../axios/settings'
+import { useState } from 'react'
+import {
+  Box,
+  FormHelperText,
+  Grid,
+  InputLabel,
+  Typography
+} from '@mui/material'
 import { stH3, stP } from '../../auth/style'
-import CustomInput from '../../auth/InputCustom'
 import { useDispatch } from 'react-redux'
 import { currentUser } from '../../../redux/authentication/operation'
 import { getToken } from '../../../utils/cookie/token'
 import { AppDispatch } from '../../../redux/store'
 import { EmailConfirmationModal } from '../../Modal-custom-btn/ModalCustomBtnEmail'
 import { UserResponse } from '../../../types'
 import { ChangeDataSchema } from '../../auth/validation'
-import { BootstrapButton } from './style'
 import { CustomSnackbar } from '../../SnackebarCustom/SnackbarCustom'
 import { useAuth } from '../../../hooks/use-auth'
+import { OutlinedInput } from '../../UI/Field'
+import { stBtn, stInput } from '../Change-password/Change-password'
+import { Button } from '../../UI/Button'
+import { SubmitHandler, UseFormRegister, useForm } from 'react-hook-form'
+import { yupResolver } from '@hookform/resolvers/yup'
+import axiosInstance from '../../../axios/settings'
 
-export type UserSubset = Pick<
-  UserResponse,
-  'email' | 'first_name' | 'last_name' | 'phone_number'
->
+type FormData = {
+  first_name: string
+  last_name: string
+  phone_number?: string | null
```

Comment:
phone_number should be only `string` it can not be `null`. it can by just empty string

Reply:
done